ipatests/test_webui/test_realmdomains.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
 # Authors:
 #   Petr Vobornik <pvoborni@redhat.com>
```

Comment:
Do we need to change this?

Reply:
We have https://github.com/freeipa/freeipa/blob/master/Contributors.txt
So... what about to delete it? @pvoborni , do you agree?